lib/views/help/how.html.erb

```diff
@@ -320,6 +320,29 @@
       response.
     </li>
   </ul>
+  <h4 id="backpaging">
+    Backpaging
+    <a href="#backpaging">#</a>
+  </h4>
+  <p>
+    Administrators can "backpage" a request thread, this means the page is
+    accessible to those who have the direct link to it, but it is not included
+    in lists of requests or search results on the site. This is similar to
+    YouTube's "unlisted" or GoogleDoc's "anyone with the link" accessibility
+    settings.
+  </p>
+  <p>
+    When a request on WhatDoTheyKnow is "backpaged" meta-tags are added to the
+    header of the request page, and linked attachments, to ask external search
+    engines not to index the material. This feature is rarely used, but can
+    help us continue to publish material we might otherwise have to remove.
```

Comment:
Maybe remove "have to" from this last line?

It's not always about "having to"; sometimes it's "want to", "prefer to", "have little technical option" etc, IYSWIM?

Reply:
This is a good point - are we happy to go with "can help us continue to publish material we might otherwise remove"?

Reply:
I'm happy with this change.

Reply:
Pushed fixup under ce14c8a - thanks folks 😃